tests/pool.ts

```diff
@@ -1,11 +1,24 @@
 import {
+  assert,
   assertEquals,
   assertThrowsAsync,
 } from "../test_deps.ts";
 import { Pool } from "../pool.ts";
 import { delay } from "../utils.ts";
 import { TEST_CONNECTION_PARAMS, DEFAULT_SETUP } from "./constants.ts";
 
+Deno.test("string pool connection", async function () {
+  const { user, password, database, hostname, port } = TEST_CONNECTION_PARAMS;
+  const pool = new Pool(
+    `postgres://${user}:${password}@${hostname}:${port}/${database}`,
+    10,
+  );
+  await pool.connect();
+  const result = await pool.query("SELECT true;");
+  await pool.end();
```

Comment:
Both these:
```
await client.release()
await pool.end()
```
?

Reply:
Surely this isn't needed? It isn't required for the other tests, right?

Reply:
Good point! 🤔